commands/run.go

```diff
@@ -39,26 +51,67 @@ var RunCmd = cli.Command{
 			return err
 		}
 
-		nodeRange, args := parseCommand(c.Args(), c.IsSet("terminator"))
-
-		if nodeRange == "" {
-			nodeRange = fmt.Sprintf("[0-%d]", len(nodes)-1)
+		var reader io.Reader
+		if c.IsSet("stdin") {
+			finfo, err := os.Stdin.Stat()
+			if err != nil {
+				return err
+			}
+			if finfo.Size() == 0 && finfo.Mode()&os.ModeNamedPipe == 0 {
+				cli.ShowCommandHelp(c, "run")
+				fmt.Println()
+				return fmt.Errorf("error: no command input and stdin is empty")
+			}
```

Comment:
What do you think about doing this up in the before and only setting `stdin` to true if it has data?

Reply:
Nice, I like it. Just pushed.